docs/source/modules/idealised_moist_phys.rst

```diff
@@ -0,0 +1,295 @@
+|``do_virtual``              |Selects whether virtual temperature is used in the vertical diffusion module.|``False``|
++----------------------------+-----------------------------------------------------------------------------+---------+
+|``roughness_moist``         |Roughness length for moisture used in surface heat exchange.                 |``0.05`` |
++----------------------------+-----------------------------------------------------------------------------+---------+
+|``roughness_mom``           |Roughness length for momentum used in surface heat exchange.                 |``0.05`` |
++----------------------------+-----------------------------------------------------------------------------+---------+
+|``roughness_heat``          |Roughness length for heat used in surface heat exchange.                     |``0.05`` |
++----------------------------+-----------------------------------------------------------------------------+---------+
+|``land_roughness_prefactor``|Multiplier on the above roughness lengths to allow land-ocean contrast.      | ``1.0`` |
++----------------------------+-----------------------------------------------------------------------------+---------+
+
+
+**Gaseous Planets**
+
+Isca currently includes options to run a Jupiter-type planet. The relevant namelist parameter for the lower boundary physics in this case is:
+
++----------------------------+-----------------------------------------------------------------------------+---------+
+| Option                     | Summary                                                                     |Default  |
++============================+=============================================================================+=========+
+|``gp_surface``              |Turns on **Schneider & Liu (2009)** prescription of lower-boundary heat flux.|``False``|
++----------------------------+-----------------------------------------------------------------------------+---------+
+
+
+Upper Level Damping
+"""""""""""""""""""
+
+At upper levels, damping may be needed to account for subgrid-scale processes that decelerate fast upper-level winds. ``damping_driver_mod`` allows Rayleigh friction to be applied at upper levels, and is currently being configured to allow gravity wave drag options. This is switched on with:
+
++----------------------------+-----------------------------------------------------------------------------+---------+
+| Option                     | Summary                                                                     |Default  |
++============================+=============================================================================+=========+
+|``do_damping``              |If true, call ``damping_driver_mod``                                         |``False``|
++----------------------------+-----------------------------------------------------------------------------+---------+
+
```

Comment:
Suggestion: The module ``damping_driver_mod`` applies Rayleigh friction at upper levels. This is switched on with: 

+----------------------------+-----------------------------------------------------------------------------+---------+
| Option                     | Summary                                                                     |Default  |
+============================+=============================================================================+=========+
|``do_damping``              |If true, call ``damping_driver_mod``                                         |``False``|
+----------------------------+-----------------------------------------------------------------------------+---------+

NB: ``damping_driver_mod`` is currently being configured to allow gravity wave drag options. 

(((otherwise it could be a bit misleading, sounding like the gravity wave drag option is switched on with do_damping)))